src/main/java/io/antmedia/rest/RestServiceBase.java

```diff
@@ -332,19 +333,27 @@ protected Result deleteBroadcast(String id) {
 		boolean stopResult = false;
 
 		if (id != null) {
-			Broadcast broacast = getDataStore().get(id);
-			stopResult = stopBroadcastInternal(broacast);
+			Broadcast broadcast = getDataStore().get(id);
+			boolean isCluster = getAppContext().containsBean(IClusterNotifier.BEAN_NAME);
+			
+			if (isCluster && !broadcast.getOriginAdress().equals(getServerSettings().getHostAddress()) && broadcast.getStatus().equals(AntMediaApplicationAdapter.BROADCAST_STATUS_BROADCASTING))
+			{
+				logger.error("Please send a Delete Broadcast request to the {} node or Delete Broadcast in a stopped broadcast.", broadcast.getOriginAdress());
+				result.setSuccess(false);
+				return result;
```

Comment:
Don't return in the middle of the method. This method is not so long. Make standard if-else block